src/components/OSCALLoader.js

```diff
@@ -56,8 +57,11 @@ export default function OSCALLoader(props) {
   };
 
   const handleReloadClick = () => {
-    setIsLoaded(false);
-    loadOscalData(oscalUrl);
+    if (isLoaded) {
+      setIsLoaded(false);
+      setContentLoaded(false);
```

Comment:
Wouldn't it make more sense to call `setLoadedStates` instead and get rid of these two function calls?